apps/nowcrm-strapi/src/index.ts

```diff
@@ -1,113 +1,115 @@
-      await createAdminRole(strapi)
-
-      const upPassword = generatePassword()
-
-      await createUsersPermissionsAdminIfNotExist(
-        strapi,
-        process.env.STRAPI_STANDART_EMAIL!,
-        upPassword
-      )
+    //     (async () => {
+//       const publishedAt = Date.now();
+
+//       // --- ensure locales etc ---
+//       await ensureLocalesAndConsents(strapi);
+
+//       // --- seed data ---
+//       await import('./bootstrap/bootstrapService')
+//         .then(({ bootstrapData }) => bootstrapData(strapi, seedData, publishedAt));
+
+//       // --- read & seed industries ---
+//       await populateStartupEntry(strapi, 'api::industry.industry', industriesData);
+
+//       // --- link entities ---
+//       const tempEmails = seedData.contact.map(c => c.email);
+//       const existingActions = await strapi.db.query('api::action.action').count({
+//         where: { contact: { email: { $in: tempEmails } } },
+//       });
+
+//       if (existingActions === 0) {
+//         await seedLinkedEntitiesForTempUsers(strapi);
+//       } else {
+//         console.log('Skipping seedLinkedEntitiesForTempUsers – already seeded.');
+//       }
+
+//       // --- setup roles, users, and tokens ---
+//       await cleanupPermissions(strapi)
+//       await createAdminRole(strapi)
+
+//       const upPassword = generatePassword()
+
+//       await createUsersPermissionsAdminIfNotExist(
+//         strapi,
+//         process.env.STRAPI_STANDART_EMAIL!,
+//         upPassword
+//       )
       
-      console.log("CRM_ADMIN_PASSWORD:", upPassword)
-
-      if (process.env.NT_ACTIVE_SERVICES?.includes('journeys')) {
-        setUpJourneysWebhook(strapi);
-      }
-
-      // check if super-admin already exists
-      const superAdminRole = await strapi.db
-        .query("admin::role")
-        .findOne({ where: { code: "strapi-super-admin" } });
-
-      const existingSuperAdmins = await strapi.db
-        .query("admin::user")
-        .count({ where: { roles: { id: superAdminRole!.id } } });
-
-      if (existingSuperAdmins === 0) {
-        const adminPassword = generatePassword();
-        await createSuperAdminUserIfNotExist(
-          strapi,
-          process.env.STRAPI_STANDART_EMAIL!,
-          adminPassword
-        );
-        console.log(`STRAPI_ADMIN_PASSWORD: ${adminPassword}`);
-      } else {
-        console.log("SuperAdmin already exists. Skipping password generation.");
-      }
-
-      await createSuperAdminTest(
-        strapi,
-        process.env.STRAPI_TEST_ADMIN_EMAIL!,
-        process.env.STRAPI_TEST_ADMIN_PASSWORD!
-      );
-
-      await createApiTokenTest(strapi);
-      await createJourneysDalComposerToken(strapi);
-      await createCrmToken(strapi); 
-
-      seedCommonEntities(strapi);
-      seedEntities(strapi);
-    })().catch((err) => {
-      console.error('Error during bootstrap:', err);
-    });
-  },
+//       console.log("CRM_ADMIN_PASSWORD:", upPassword)
```

Comment:
🛑 **Gitleaks** has detected a secret with rule-id `generic-secret` in commit fb238356ff1eed106a9ddb761791dc554a89b095.
If this secret is a _true_ positive, please rotate the secret ASAP.

If this secret is a _false_ positive, you can add the fingerprint below to your `.gitleaksignore` file and commit the change to this branch.

```
echo fb238356ff1eed106a9ddb761791dc554a89b095:apps/nowcrm-strapi/src/index.ts:generic-secret:72 >> .gitleaksignore
```